plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -170,9 +170,6 @@ class OptunaSweeperConf:
     # Total number of function evaluations
     n_trials: int = 20
 
-    # Number of parallel workers
-    n_jobs: int = 2
-
```

Comment:
This is a breaking change.

Instead of changing the API for `OptunaSweeperConf`, what if we call `ConfigStore.store` twice? We can do something like `store(node=OptunaSweeperConfV2, name="optuna_v2")` for the new API and `store(node=OptunaSweeperConf, name="optuna")` for backward compatibility.